packages/tables/backend/sirius-components-tables/src/main/java/org/eclipse/sirius/components/tables/components/TableComponent.java

```diff
@@ -56,6 +58,16 @@ public Element render() {
         TableRenderingCache cache = new TableRenderingCache();
         ITableElementRequestor tableElementRequestor = new TableElementRequestor();
 
+        AtomicReference<String> globalFilter = new AtomicReference<>("");
+        if (this.props.globalFilter() == null) {
+            optionalPreviousTable.ifPresent(previousTable -> {
+                globalFilter.set(previousTable.getGlobalFilter());
+                variableManager.put(TableRenderer.GLOBAL_FILTER_DATA, previousTable.getGlobalFilter());
```

Comment:
Don't conditionally put variables, they should always be there but their value can be null otherwise AQL expressions cannot compile